src/lib/zod/index.ts

```diff
@@ -258,6 +258,12 @@ export const updateCircleInput = z
   })
   .strict();
 
+export const restoreCoordinapeInput = z
```

Comment:
@topocount @zashton IMO these should be co-located with handlers when they're not used in multiple places; what do you think?